tests/unit_tests/test_geometry.py

```diff
@@ -43,6 +43,9 @@ def test_volume(run_in_tmpdir, uo2):
         nucs = set(domain.get_nuclide_densities())
         assert not nucs ^ {'U235', 'O16'}
 
+        # make sure standard deviation is also set from the calculation
+        assert domain.volume_std is not None
```

Comment:
```suggestion
        assert isinstance(domain.volume_std, float)
```
Wondering if it is worth checking we have a float here instead which I guess also checks if it is not None